app/protocols/fetch-to-handler.js

```diff
@@ -19,11 +20,12 @@ module.exports = function fetchToHandler (getFetch, session) {
 
   async function * readBody (body) {
     for (const chunk of body) {
-      if (chunk.blobUUID) {
-        const data = await session.getBlobData(chunk.blobUUID)
-        yield data
-      } else if (chunk.bytes) {
+      if (chunk.bytes) {
         yield await Promise.resolve(chunk.bytes)
+      } else if (chunk.blobUUID) {
+        yield await session.getBlobData(chunk.blobUUID)
+      } else if(chunk.file){
+        yield await fs.readFile(chunk.file)
```

Comment:
Could we use `yield* Readable.from(fs.createReadStream())` instead here so that the data can be streamed in?

Also, what's the best way to test this?

Reply:
fixed.

as far as testing, i will get to it and create a test. maybe we can create an object filled with data a `File` object would have and then go from there.

Reply:
@RangerMauve   just realized that agregore doesn't have a test file. i tested myself by uploading an entire vuejs test site. but that was me actually using my changes on agregore, not a programmatic test. how would you suggest to test this?